ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
+  "contact_unlocked": "Nowy kontrakt został odblokowany.",
+  "equipment_unknown": "Błąd: Nieznany ekwipunek.", //Review and context needed here!
+  "equipment_type_unknown": "Błąd: Nieznany typ ekwipunku.", //Review and context needed here!
+  "equipment_space_already_occupied": "Błąd: To miejsce zostało już zajęte.", //Review and context needed here!
+  "equipment_missing_resources": "Błąd: Za mała ilość zasobów aby wyprodukować ten rodzaj ekwipunku.", //Review and context needed here!
```

Comment:
Same, error but when you don't have enough resources to build it (here, not enough money usually, but it won't always be the case)